routes/home.js

```diff
@@ -31,7 +31,9 @@ module.exports = async (req, res) => {
       calendarButton: true,
       filterButtons: true,
       english: req.params.lang !== 'ar',
-      arabic: req.params.lang === 'ar'
+      currentDate: new Date().toISOString().split('T')[0],
+      arabic: req.params.lang === 'ar',
+      numberOfEvents: events.length
```

Comment:
This could probably be a `length` helper so there's less to pass in.

Reply:
that would involve creating a file and passing it to the hbs config. i feel like it's kinda same no?